sway/commands/output/color_profile.c

```diff
@@ -0,0 +1,103 @@
+#define _XOPEN_SOURCE 700
```

Comment:
Can we `#undef _POSIX_C_SOURCE` if we need `_XOPEN_SOURCE`? Otherwise we provide two feature test macros and they potentially conflict (in which case what we get is implementation-defined).